testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java

```diff
@@ -381,7 +431,19 @@ public static void loginToApp(String username, String password, int expectedStat
                 if (expectedText != null) {
                     String responseString = new BasicResponseHandler().handleResponse(afterLoginClickResponse);
                     assertTrue("Unexpected result " + responseString, responseString.contains(expectedText));
+                    if (expectedScope != null){
+                        assertTrue(context.toString().contains("scope=" + expectedScope));
+                        if(expectedScope.contains("profile")){
+                            assertTrue(responseString.contains("profile: " + KeycloakConfiguration.ALICE_FIRST_NAME + " " + KeycloakConfiguration.ALICE_LAST_NAME));
+                        }
+                        if(expectedScope.contains("email")){
```

Comment:
Same here.

Reply:
Sorry, I think I dropped some changes when rebasing. 
Should be good now. Thanks for catching it.